### DISCUSSION TITLE
How to change primary colors?

### DISCUSSION THREAD

**Discussion by doniyorsaidmurodov** [primefaces/community#2982] [PrimeNG]
Here by default it uses emerald color. How I can change primary color?

constructor(private config: PrimeNGConfig) {
    this.config.theme.set({
      preset: Aura,
      options: {
        darkModeSelector: '.dark'
      }
    });
    this.config.ripple.set(true);
  }

**Comment by qwadrox** [comment]
@doniyorsaidmurodov Hi!

You can find videos on YouTube on how to style in case the documentation is insufficient.

For instance: 
https://www.youtube.com/watch?v=J3KFw5sih98
